README.md

```
@@ -150,6 +150,48 @@ volumes:
+**Docker:** Mount your script directory into the container and point `post_script` at the in-container path:
+
+```yaml
+volumes:
+  - ./scripts:/scripts:ro
+```
+
+## Tick Notices
+
+At the end of each check cycle, StreamDL logs a wait line followed by any actionable notices collected during that tick:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)